app/src/App.vue

```diff
@@ -42,6 +57,27 @@ watch(
     updateUserInput.value.isValid = isAddress(newVal)
   }
 )
+watch(withdrawError, () => {
```

Comment:
You can consolidate the monitoring of related variables by using a single watch function. Makes it more readable in my opinion.

Reply:
use watchEffect like this?

```typescript
watchEffect(() => {
  if (withdrawError.value) {
    toastStore.show(
      ToastType.Error,
      withdrawError.value.reason ? withdrawError.value.reason : 'Failed to withdraw tips'
    )
  }
  if (withdrawSuccess.value) {
    toastStore.show(ToastType.Success, 'Tips withdrawn successfully')
  }
  if (balanceError.value) {
    toastStore.show(
      ToastType.Error,
      balanceError.value.reason ? balanceError.value.reason : 'Failed to get balance'
    )
  }
})
```

Reply:
This is an example from AddTeamCard.vue. Something similar to this.
```typescript
watch(
  [() => props.showAddTeamModal, team, showAddTeamModal],
  ([newshowAddTeamModal, newFormDataTeam]) => {
    emits('updateAddTeamModal', newFormDataTeam)
    showAddTeamModal.value = newshowAddTeamModal
  },
  { deep: true }
)
```